corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -48,16 +48,20 @@ hqDefine("cloudcare/js/formplayer/menus/api", [
                 menus;
 
             $.when(AppsAPI.getAppEntities()).done(function (appCollection) {
+                const app = appCollection.find(function (app) {
+                    if (app.id && app.id === params.appId) {
```

Comment:
I think this is only needed temporarily right? It's for the transient period when this change is deployed but someone is in the middle of using webapps so appId in the URL is still defined as the build Id

If so, worth adding the comment that this conditional can be deleted after deploy.